src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -0,0 +1,225 @@
+                    :error-messages="errors"
+                    type="password"
+                    v-model="password"
+                    @keyup.enter="submit"
+                  ></v-text-field>
+                </ValidationProvider>
+              </ValidationObserver>
+            </v-form>
+          </v-card-text>
+          <v-card-actions class="px-4">
+            <xrd-button
+              id="submit-button"
+              color="primary"
+              gradient
+              block
+              large
+              @click="submit"
+              :min_width="120"
+              rounded
+              :disabled="isDisabled"
+              :loading="loading"
+              >{{ $t('login.logIn') }}</xrd-button
+            >
+          </v-card-actions>
+        </v-card>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue, { VueConstructor } from 'vue';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+
+export default (Vue as VueConstructor<
+  Vue & {
+    $refs: {
+      form: InstanceType<typeof ValidationObserver>;
+    };
+  }
+>).extend({
+  name: 'login',
+  components: {
+    ValidationProvider,
+    ValidationObserver,
+  },
+  data() {
+    return {
+      loading: false as boolean,
+      username: '' as string,
+      password: '' as string,
+    };
+  },
+  computed: {
+    isDisabled() {
+      if (
+        this.username.length < 1 ||
+        this.password.length < 1 ||
+        this.loading
+      ) {
+        return true;
+      }
+      return false;
+    },
+  },
+  methods: {
+    async submit() {
+      // Validate inputs
+
+      const isValid = await this.$refs.form.validate();
+
+      if (!isValid) {
+        return;
+      }
+
+      this.$refs.form.reset();
+      this.loading = true;
+
+      this.$router.replace({
+        name: 'base',
+      });
+    },
+    fetchUserData() {
+      // do something later
+    },
+
+    fetchInitializationData() {
+      // do something later
+    },
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@import '~styles/colors';
+
+.graphics {
+  height: 100%;
+  width: 40%;
+  max-width: 576px; // width of the backround image
+  background-image: url('../assets/background.png');
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.login-view-wrap {
+  background-color: white;
+  padding: 0;
+}
+
+.title-wrap {
+  display: flex;
+  flex-direction: column;
+}
+
+.login-form-toolbar {
+  background-color: white;
+  margin-bottom: 30px;
+  padding-left: 0;
+}
+
+.login-form-toolbar-title {
+  margin-left: 0;
+  color: #252121;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 54px;
+}
+
+.sub-title {
+  font-style: normal;
+  font-weight: normal;
+  font-size: $XRoad-DefaultFontSize;
+  line-height: 19px;
+}
+
+.set-width {
+  max-width: 420px;
+}
```

Comment:
See if the css-class hierarchy could be utilized

Reply:
Fixed